composables/numberFormatter.ts

```diff
@@ -0,0 +1,3 @@
+export function useNumberFormatter() {
+  return Intl.NumberFormat('en-GB', {}).format
```

Comment:
could we define this outside of the function?

Reply:
Good point, thanks. How does this look? https://github.com/nuxt/nuxters/compare/6dbcb0f0204fc1ffc0580b60785301b2accd3e6e..17fe8c5991ced83030dbd6d2f97226d5ef0547b3